contributors/guide/contributor-cheatsheet/README-ko.md

```diff
@@ -0,0 +1,398 @@
+# 쿠버네티스 컨트리뷰터 참고 자료(Cheat Sheet)
+
+쿠버네티스에 기여할 때, 일반적인 자원, 팁, 요령과
+쿠버네티스 프로젝트 내에서 사용되는 일반적인 모범 사례의 리스트이다.
+이 문서는 GitHub 컨트리뷰션 경험을 더 좋게 만들기 위한 "TL;DR" 또는
+유용한 정보의 빠른 레퍼런스이다.
+
+**목차**
+- [도움이 되는 자원](#도움이-되는-자원)
+  - [시작하기](#시작하기)
+  - [SIG 및 기타 그룹](#SIG-및-기타-그룹)
+  - [커뮤니티](#커뮤니티)
+  - [중요한 이메일 별칭](#중요한-이메일-별칭)
+  - [작업 흐름](#작업-흐름)
+  - [테스트](#테스트)
+  - [다른 유용한 링크](#다른-유용한-링크)
+- [GitHub에서 효과적으로 의사 소통하기](#GitHub에서-효과적으로-의사-소통하기)
+  - [서로에게 훌륭한 방법](#서로에게-훌륭한-방법)
+    - [좋은/나쁜 의사 소통 예시](#좋은/나쁜-의사-소통-예시)
+- [컨트리뷰션 전송](#컨트리뷰션-전송)
+  - [CLA 서명](#CLA-서명)
+  - [이슈의 오픈 및 응답](#이슈의-오픈-및-응답)
+    - [이슈 생성](#이슈-생성)
+    - [이슈 응답](#이슈-응답)
+  - [풀 리퀘스트 오픈](#풀-리퀘스트-오픈)
+    - [풀 리퀘스트 생성](#풀-리퀘스트-생성)
+    - [PR 설명 예시](#PR-설명-예시)
+    - [풀 리퀘스트 문제 해결](#풀-리퀘스트-문제-해결)
+  - [레이블](#레이블)
+- [로컬에서 작업](#로컬에서-작업)
+  - [브랜치 전략](#브랜치-전략)
+    - [Upstream 추가](#Upstream-추가)
+    - [Fork를 동기화 상태로 유지하기](#Fork를-동기화-상태로-유지하기)
+  - [스쿼시 커밋](#스쿼시-커밋)
+
+---
+
+## 도움이 되는 자원
+
+### 시작하기
+
+- [컨트리뷰터 가이드] - 어떻게 쿠버네티스 프로젝트에 기여할 수 있는지에 대한
+  가이드.
```

Comment:
Not too much sure of your opinion, but my general suggestion to use period(".") is like the followings:
- Use a period at the end of each complete sentence.
- Do not use a period for incomplete sentences.

How do you think of my suggestion?